module/apmhttp/ignorer.go

```diff
@@ -53,6 +56,15 @@ func DefaultServerRequestIgnorer() RequestIgnorerFunc {
 	return defaultServerRequestIgnorer
 }
 
+// DynamicServerRequestIgnorer returns the RequestIgnorer to use in
+// handlers. The list of wildcard patterns comes from central config
+func DynamicServerRequestIgnorer(t *apm.Tracer) RequestIgnorerFunc {
```

Comment:
```suggestion
func NewDynamicServerRequestIgnorer(t *apm.Tracer) RequestIgnorerFunc {
```

Let's align the naming with the other functions below, since DefaultServerRequestIgnorer will be going away eventually.

Reply:
In fact, if you implement the suggestion above about changing Tracer.IgnoredTransactionURLMatchers to IgnoredTransactionURL, then we can just pass `tracer.IgnoredTransactionURL` directly in as it will satisfy the function signature -- no need for (New)DynamicServerRequestIgnorer then.

Reply:
I realise now that RequestIgnorerFunc takes an *http.Request, whereas IgnoredTransactionURL takes a URL, so my second comment doesn't work. But let's please align the naming, and add `New` to the front.